src/dialects/mysql/schema/columncompiler.js

```diff
@@ -7,6 +7,17 @@ import * as helpers from '../../../helpers';
 
 import { assign } from 'lodash'
 
+function supportsPreciseTimestamps(client) {
+  if (!client.version) {
```

Comment:
I just realised that actually every mysql user needs to set that variable to get rid of the warning 😬 So I think the way to go there is to change a error message a bit to be like:

'To get rid of this warning one should specify mysql dialect flavor in the knex configuration. Currently flavor defaults to 5.5, but in future releases it will default to 5.6 that supports high precision timestamps. See http://knexjs.org/#Schema-timestamps for more information."

And then just remove the warning in some future release and change the default.

Reply:
Ok, will do.

Reply:
Updated.